test/web/10-VerifiableCredentialStore.js

```diff
@@ -81,13 +81,14 @@ describe('VerifiableCredentialStore', () => {
     meta.updated.should.be.a('number');
   });
 
-  it('should not overwrite an existing credential with default mutator', async () => {
+  it('should not overwrite an existing credential with default mutator' +
+    '', async () => {
```

Comment:
There's got to be a better way to wrap lines without indenting the entire block.

Reply:
Sorry about the linting errors here that came about in #24 . For whatever reason vscodes eslint plugin wasn't giving me any warnings...